analytics/README.md

```diff
@@ -16,11 +16,42 @@ It will strain the network and might exceed some Github rate limits. Running scr
 ### Prerequisites
 - [Node](https://nodejs.org/en/)
 - [Github cli](https://cli.github.com/)
+- Github account
+- [Github PAT for authentication](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token)
 - [React-scanner](https://github.com/moroshko/react-scanner) as globally or locally installed
 
 ### Running the script
+Set your Github PAT to environment variable called `GITHUB_TOKEN` or inject it into the command below.
 To collect analytics from a subset of found repositories (for testing purposes), run:
-```OFFSET=0 LIMIT=1 node run-analytics.js```
+```OFFSET=0 LIMIT=1 TOKEN=$GITHUB_TOKEN node run-analytics.js```
```

Comment:
Should we use TOKEN or GITHUB_TOKEN everywhere?
In this script the env variable is TOKEN and in the bash script it is GITHUB_TOKEN